src/TestStack.White/InputDevices/Keyboard.cs

```diff
@@ -1,193 +1,230 @@
 using TestStack.White.WindowsAPI;
 using Action = TestStack.White.UIItems.Actions.Action;
 
 namespace TestStack.White.InputDevices
 {
-    //BUG: KeysConverter
+    // BUG: KeysConverter
     /// <summary>
     /// Represents Keyboard attachment to the machine.
     /// </summary>
     public class Keyboard : IKeyboard
     {
-        private static readonly List<KeyboardInput.SpecialKeys> scanCodeDependent = new List<KeyboardInput.SpecialKeys>
-                                                                               {
-                                                                                   KeyboardInput.SpecialKeys.RIGHT_ALT,
-                                                                                   KeyboardInput.SpecialKeys.INSERT,
-                                                                                   KeyboardInput.SpecialKeys.DELETE,
-                                                                                   KeyboardInput.SpecialKeys.LEFT,
-                                                                                   KeyboardInput.SpecialKeys.HOME,
-                                                                                   KeyboardInput.SpecialKeys.END,
-                                                                                   KeyboardInput.SpecialKeys.UP,
-                                                                                   KeyboardInput.SpecialKeys.DOWN,
-                                                                                   KeyboardInput.SpecialKeys.PAGEUP,
-                                                                                   KeyboardInput.SpecialKeys.PAGEDOWN,
-                                                                                   KeyboardInput.SpecialKeys.RIGHT,
-                                                                                   KeyboardInput.SpecialKeys.LWIN,
-                                                                                   KeyboardInput.SpecialKeys.RWIN
-                                                                               };
-
         private readonly List<KeyboardInput.SpecialKeys> heldKeys = new List<KeyboardInput.SpecialKeys>();
+        protected readonly List<int> KeysHeld = new List<int>();
 
         /// <summary>
-        /// Use Window.Keyboard method to get handle to the Keyboard. Keyboard instance got using this method would not wait while the application
-        /// is busy.
+        /// Use Window.Keyboard method to get handle to the Keyboard. 
+        /// Keyboard instance got using this method would not wait while the application is busy.
         /// </summary>
-        public static readonly Keyboard Instance = new Keyboard();
-
-        private readonly List<int> keysHeld = new List<int>();
+        [Obsolete("Obsolete. Create an instance where it is needed.")]
+        public static readonly IKeyboard Instance = new Keyboard();
```

Comment:
I don't mind having a default instance available. It saves on allocations and the majority of the time you want to access this as static?

Or do you think the majority access they mouse/keyboard via UIItems?